app/k9mail/build.gradle

```diff
@@ -36,7 +36,7 @@ dependencies {
 
 android {
     compileSdkVersion buildConfig.compileSdk
-    buildToolsVersion buildConfig.buildTools
```

Comment:
Please don't change this line. Changing the value of the variable is sufficient.